lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -452,7 +452,8 @@ def send_bulk(data)
       response = client.bulk body: data
       if response['errors']
         @error.handle_error(response)
-        log.error "Could not push log to Elasticsearch: #{response}"
+        log.error "Could not push log to Elasticsearch with Bulk Request. Add '@log_level debug' to your config to see full response."
+        log.on_debug { log.debug "Failed bulk response: #{response}" }
```

Comment:
Do we need this here?  Looks like the handle_error method already logs the response in the error cases
I'm not sure we need the log.error here either